Formula/c/cekit.rb

```diff
@@ -0,0 +1,25 @@
+class Cekit < Formula
+  include Language::Python::Virtualenv
+
+  desc "Container Evolution Kit"
+  homepage "https://cekit.io"
+  url "https://files.pythonhosted.org/packages/c4/cc/a948fb7fc87bb974f55b37da024734166d4981a4f43958f1efc84d2b1e3d/cekit-4.10.0.tar.gz"
+  sha256 "83909f20463073aeb2d5f56295b7446c39ea4605fe8640ccfb521c618c5663e6"
+  license "MIT"
+
+  depends_on "python-click"
```

Comment:
this needs more than just click and pyyaml, no? `brew update-python-resources cekit` will automatically generate all the `resource`'s for you

```console
$ pipgrip --tree cekit
cekit (4.10.0)
├── click>=6.7 (8.1.7)
├── colorlog>=2.10.0 (6.8.2)
├── jinja2>=2.7 (3.1.3)
│   └── markupsafe>=2.0 (2.1.5)
├── odcs>=0.2.10 (0.8.0)
├── packaging>=19.0 (23.2)
├── pykwalify>=1.6.0 (1.8.0)
│   ├── docopt>=0.6.2 (0.6.2)
│   ├── python-dateutil>=2.8.0 (2.8.2)
│   │   └── six>=1.5 (1.16.0)
│   └── ruamel-yaml>=0.16.0 (0.18.5)
│       └── ruamel-yaml-clib>=0.2.7 (0.2.8)
└── pyyaml>=3.10 (6.0.1)